packages/anvil-embed-frame/src/index.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+class AnvilEmbedFrame extends React.Component {
+  constructor (props) {
+    super(props)
+    this.iframeRef = React.createRef()
+  }
+
+  componentDidMount () {
+    const { scroll } = this.props
+    if (scroll) this.iframeRef.current.scrollIntoView({ behavior: scroll })
+    window.addEventListener('message', this.handleEvent)
+  }
+
+  componentWillUnmount () {
+    window.removeEventListener('message', this.handleEvent)
+  }
+
+  handleEvent = ({ origin, data }) => {
+    const { anvilURL, onEvent } = this.props
+    if (anvilURL !== origin) return
+    if (typeof data === 'object') {
+      onEvent(data)
+    }
+  }
+
+  render () {
+    const { iframeURL, onEvent, anvilURL, enableDefaultStyles, scroll, ...others } = this.props
+    return (
+      <iframe
+        id="anvil-embed-frame"
+        name="Anvil Embed Frame"
+        title="Anvil Embed Frame"
+        {...others} // props above may be overriden
+        src={iframeURL}
+        ref={this.iframeRef}
+        style={enableDefaultStyles
+          ? {
+              width: '80vw',
+              height: '85vh',
+              maxWidth: '1200px',
+              borderStyle: 'groove',
+            }
+          : undefined}
```

Comment:
I'd put this above the `...others` so people can specify their own styles if they want